assembl/static2/js/app/components/login/legalForm.jsx

```diff
@@ -0,0 +1,86 @@
+// @noflow
+import * as React from 'react';
+import { Translate } from 'react-redux-i18n';
+import { Modal, Button } from 'react-bootstrap';
+import { closeModal } from '../../utils/utilityManager';
+
+type LegalFormProps = {
+  checked: boolean,
+  handleAcceptButton: () => void,
+  style?: Object
+};
+
+type LegalFormState = {
```

Comment:
`Props` and `State`

Reply:
In storybook we have to export the types, so for future components following this new workflow we'll have to call the types this way. Therefore I thought we should already apply it. No?

Reply:
You can still use `export type Props = {}` and when you import it use `import { type Props as LegalFormProps } from './legalForm'`